Source/diablo.cpp

```diff
@@ -464,10 +464,9 @@ void diablo_parse_flags(char *args)
 			case 'w':
 				debug_mode_key_w = 1;
 				break;
-			case 'x':
-				fullscreen = 0;
-				break;
-			}
```

Comment:
Looks like you broke the switch case braces, maybe fix your looking for is to disable parameter passing all together? 

Can you see what parameters are being passed to the program on launch?

If you need to prevent parameters from being sent to the game then please do this in /SourceX/main.cpp instead of changing files in /Source.